### PR TITLE
Update feed sync data configuration section

### PR DESCRIPTION
**Pull request by nightfurys**
Signed-off-by: Swathi Gangisetty <swathi@anchore.com>

### CODE REVIEW

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 1]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
 ```
-### Feed Settings - Grype
-By default, Anchore Enterprise syncs the Grype feed.
-The Grype feed will contain the records from all the other groups. It is not possible to include or exclude groups from the Grype feed.
-VulnDB and Microsoft feed data is not yet supported for the tech preview Grype vulnerability scanner. Those feeds will not be synced.
+### Feed Data Settings
 
+Feed data configuration is set in the config.yaml file used by policy engine service. The `services.policy_engine.vulnerabilities.sync.data` section
+of the configuration file controls the behavior of data to be synced. In addition, the data groups that can be synced depend on the `services.policy_engine.vulnerabilities.provider`, 
+further explained below.
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:24]:
```suggestion
and are explained in detail in the following sections.
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 2]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
+Feed data configuration is set in the config.yaml file used by policy engine service. The `services.policy_engine.vulnerabilities.sync.data` section
+of the configuration file controls the behavior of data to be synced. In addition, the data groups that can be synced depend on the `services.policy_engine.vulnerabilities.provider`, 
+further explained below.      
 
-### Feed Settings - Legacy
-
-Feed sync configuration is set in the config.yaml file used by policy engine service. The `services.policy_engine.vulnerabilities.sync.data` section
-of the configuration file in the policy engine's container controls the behavior of feed syncs done by that particular container. Note that the location
-and format of this config data changed slightly in Anchore Engine 0.10 to reflect some internal refactoring.  Ensure this config is synchronized between
+Note that the location and format of this config data changed slightly in Anchore Engine 0.10 to reflect some internal refactoring.  Ensure this config is synchronized between
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:26]:
```suggestion
Note: The location and format of the config data changed slightly in Anchore Engine 0.10 to reflect some internal refactoring.  Ensure this 
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 3]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
 containers if you are running more than one policy engine. This is usually handled for you by Helm Charts on Kubernetes, for example.
 
-Anchore Enterprise will default to downloading feed data from Anchore's feed service hosted at https://ancho.re/v1/service/feeds and running in AWS in the
-us-west-2 region.
+#### Grype  
 
-For legacy, by default, Anchore Enterprise will only sync the non-grype feeds enabled in the config section shown below. Setting additional feed types to true or false will
-enable or disable, respectively, synchronization of the specified feed.
+By default, Anchore Enterprise is configured with `grype` as the `services.policy_engine.vulnerabilities.provider` and `grypedb` feed group enabled. 
+The `grypedb` feed group syncs a single grype database to the policy engine.
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:32]:
```suggestion
The `grypedb` feed group syncs a single Grype database to the policy engine. 
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 4]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
 
-Anchore Enterprise will default to downloading feed data from Anchore's feed service hosted at https://ancho.re/v1/service/feeds and running in AWS in the
-us-west-2 region.
+#### Grype  
 
-For legacy, by default, Anchore Enterprise will only sync the non-grype feeds enabled in the config section shown below. Setting additional feed types to true or false will
-enable or disable, respectively, synchronization of the specified feed.
+By default, Anchore Enterprise is configured with `grype` as the `services.policy_engine.vulnerabilities.provider` and `grypedb` feed group enabled. 
+The `grypedb` feed group syncs a single grype database to the policy engine. 
+A grype database contains data spanning multiple groups. Due to this encapsulation, it is not possible to enable or disable individual feed groups
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:33]:
```suggestion
A Grype database contains data that spans multiple groups. Due to this encapsulation, it is not possible to enable or disable individual feed groups
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 5]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
 
+Anchore Enterprise will default to downloading the feed group from a publicly accessible url maintained by Grype https://toolbox-data.anchore.io/grype/databases/listing.json.
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:35]:
```suggestion
Anchore Enterprise will default to downloading the feed group from a publicly accessible URL maintained by Grype https://toolbox-data.anchore.io/grype/databases/listing.json. 
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 6]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
+#### Grype  
 
-For legacy, by default, Anchore Enterprise will only sync the non-grype feeds enabled in the config section shown below. Setting additional feed types to true or false will
-enable or disable, respectively, synchronization of the specified feed.
+By default, Anchore Enterprise is configured with `grype` as the `services.policy_engine.vulnerabilities.provider` and `grypedb` feed group enabled. 
+The `grypedb` feed group syncs a single grype database to the policy engine. 
+A grype database contains data spanning multiple groups. Due to this encapsulation, it is not possible to enable or disable individual feed groups
 
+Anchore Enterprise will default to downloading the feed group from a publicly accessible url maintained by Grype https://toolbox-data.anchore.io/grype/databases/listing.json. 
+The grype database available from this endpoint does not include third party/proprietary groups such as VulnDB and MSRC.
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:36]:
```suggestion
The Grype database available from this endpoint does not include third-party/proprietary groups such as VulnDB and MSRC. 
```

File: content/docs/using/cli_usage/feeds/feed_configuration.md [hunk 7]
```diff
@@ -17,38 +17,65 @@ For most users the only configuration option that is typically updated is the fe
...
+If `services.policy_engine.vulnerabilities.provider` is set to `legacy`, Anchore Enterprise will try to sync the following groups 
+- `vulnerabilities` (enabled by default)
+- `nvdv2` (enabled by default)
+- `github` (enabled by default)
+- `vulndb `
+- `msrc`
+- `packages`
+
+Anchore Enterprise will default to downloading feed groups from Anchore's feed service hosted at https://ancho.re/v1/service/feeds and running in AWS in the
+us-west-2 region. This does not include third party/proprietary groups such as VulnDB and MSRC.
```
Comment (anchoredeanlappi) [inline content/docs/using/cli_usage/feeds/feed_configuration.md:65]:
```suggestion
us-west-2 region. This does not include third-party/proprietary groups such as VulnDB and MSRC. 
```

**Review by anchoredeanlappi — CHANGES_REQUESTED** [review]
Just a few minor edits. Thank you!

**Review by zhill — APPROVED** [review]

**Pull request merged**